AUDIO/SetRadioStationDisabled.md

```diff
@@ -0,0 +1,70 @@
+---
+ns: HUD
+aliases: N_0x94f2e83ead7e6b82
+---
+## _SET_RADIO_STATION_DISABLED
+
+```c
+// 0x94F2E83EAD7E6B82
+BOOL _SET_RADIO_STATION_DISABLED(char StationName, BOOL Toggle);
+```
+
+Disables the radio station from the radio wheel.  
+Note: You cannot disable LSUR, WCTR and Blaine County Radio.  
+
+List of known station names;
+```
+RADIO_01_CLASS_ROCK  
+RADIO_02_POP  
+RADIO_03_HIPHOP_NEW  
+RADIO_04_PUNK  
+RADIO_05_TALK_01  
+RADIO_06_COUNTRY  
+RADIO_07_DANCE_01  
+RADIO_08_MEXICAN  
+RADIO_09_HIPHOP_OLD  
+RADIO_12_REGGAE  
+RADIO_13_JAZZ  
+RADIO_14_DANCE_02  
+RADIO_15_MOTOWN  
+RADIO_20_THELAB  
+RADIO_16_SILVERLAKE  
+RADIO_17_FUNK  
+RADIO_18_90S_ROCK  
+RADIO_21_DLC_XM17  
+RADIO_22_DLC_BATTLE_MIX1_RADIO  
+RADIO_19_USER  
+RADIO_30_COMMUNITYSLOT  
+RADIO_31_COMMUNITYSLOT  
+HIDDEN_RADIO_AMBIENT_TV  
+HIDDEN_RADIO_AMBIENT_TV_BRIGHT  
+HIDDEN_RADIO_01_CLASS_ROCK  
+HIDDEN_RADIO_ADVERTS  
+HIDDEN_RADIO_02_POP  
+HIDDEN_RADIO_03_HIPHOP_NEW  
+HIDDEN_RADIO_04_PUNK  
+HIDDEN_RADIO_06_COUNTRY  
+HIDDEN_RADIO_07_DANCE_01  
+HIDDEN_RADIO_09_HIPHOP_OLD  
+HIDDEN_RADIO_12_REGGAE  
+HIDDEN_RADIO_15_MOTOWN  
+HIDDEN_RADIO_16_SILVERLAKE  
+RADIO_22_DLC_BATTLE_MIX1_CLUB  
+HIDDEN_RADIO_STRIP_CLUB  
+DLC_BATTLE_MIX1_CLUB_PRIV  
+HIDDEN_RADIO_BIKER_CLASSIC_ROCK  
+DLC_BATTLE_MIX2_CLUB_PRIV  
+HIDDEN_RADIO_BIKER_MODERN_ROCK  
+RADIO_23_DLC_BATTLE_MIX2_CLUB  
+RADIO_25_DLC_BATTLE_MIX4_CLUB  
+DLC_BATTLE_MIX3_CLUB_PRIV  
+RADIO_26_DLC_BATTLE_CLUB_WARMUP  
+HIDDEN_RADIO_BIKER_PUNK  
+RADIO_24_DLC_BATTLE_MIX3_CLUB  
+DLC_BATTLE_MIX4_CLUB_PRIV  
+HIDDEN_RADIO_BIKER_HIP_HOP  
+```
+
+## Parameters
+* **StationName**: The name of the radio station
```

Comment:
Lowercase here as well. Also, since you made it return `BOOL`, please add a `Return value` section.